### DISCUSSION TITLE
Icon request: Trophy or "winner"

### DISCUSSION THREAD

**Discussion by haugstrup** [tailwindlabs/heroicons#385] [Ideas]
The emoji set has both a trophy 🏆 and a set of medals 🥇 🥈 🥉 
In icon shape I think the **trophy** is the most recognizable shape.

**Comment by DoctorDerek** [comment]
Friendly bump, would love something similar to this icon😀

https://fontawesome.com/v5.15/icons/trophy
![image](https://user-images.githubusercontent.com/761231/125538268-7a25520a-887b-4c9c-b09e-4727b06896ef.png)

and this
https://fontawesome.com/v5.15/icons/medal
![image](https://user-images.githubusercontent.com/761231/125538292-240bcbe7-97e2-4ebc-b445-7ac7d4fdf99d.png)

**Comment by larkarvin** [comment]
any update on this?

**Reply by thelfensdrfer** [reply]
It's named "trophy" and is implemented since 2021.
